From 277cafcd424b73c41351b80f5bb3b7a3ba93803f Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Thu, 11 Jan 2018 15:46:12 +0000
Subject: [PATCH 1/2] data-ipa-cfg-mgr: Disable all the debugging

 * Same logic as in commit 5463a591007ae63ba3720ed295eac5ef19cc2ce0.

Change-Id: I7ca76b868a4678fb5d8280928f90160973e51871
---
 ipanat/src/Android.mk | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/ipanat/src/Android.mk b/ipanat/src/Android.mk
index 24ec7cf..af786eb 100644
--- a/ipanat/src/Android.mk
+++ b/ipanat/src/Android.mk
@@ -20,7 +20,10 @@ LOCAL_EXPORT_C_INCLUDE_DIRS := $(LOCAL_PATH)/../inc
 LOCAL_VENDOR_MODULE := true
 LOCAL_MODULE_PATH_64 := $(TARGET_OUT_VENDOR)/lib64
 LOCAL_MODULE_PATH_32 := $(TARGET_OUT_VENDOR)/lib
-LOCAL_CFLAGS := -DDEBUG -Wall -Werror
+ifneq (,$(filter eng, $(TARGET_BUILD_VARIANT)))
+LOCAL_CFLAGS := -DDEBUG
+endif
+LOCAL_CFLAGS += -Wall -Werror
 LOCAL_CFLAGS += -DFEATURE_IPA_ANDROID
 LOCAL_MODULE := libipanat
 LOCAL_MODULE_TAGS := optional
-- 
2.17.1

